index.html

```diff
@@ -85,8 +87,8 @@ <h2>OUR SERVICES</h2>
         <h3 class="trending-now">TRENDING NOW</h3>
         <div class="line"></div>
         <div class="products-container"></div>
-        <div class="show-more"><button type="submit" id="more">MORE<img class="arrow" src="img/products/arrow.png" alt="arrow"></button></div>
-        <div class="show-more"><button type="submit" id="less">LESS<img class="arrow" src="img/products/arrow-reverse.png" alt="arrow"></button></div>
+        <div class="show-more"><button type="submit" id="more">MORE<img class="arrow" src="img/products/arrow-reverse.png" alt="arrow"></button></div>
+        <div class="show-more"><button type="submit" id="less">LESS<img class="arrow" src="img/products/arrow.png" alt="arrow"></button></div>
```

Comment:
- [ ] (OPTIONAL) Also kindly fix the functionality of your `more` and `less` buttons as it misfunctions when switching to and from the mobile view.
- Your design:
![Animation](https://user-images.githubusercontent.com/85113567/169015975-4b84b590-ea38-47f3-a3b4-ab2be6650ad1.gif)